### ISSUE TITLE
Node selection outline not shown in editor when node is locked

### ISSUE THREAD

**Issue by EricEzaM** [godotengine/godot#37892]
**Godot version:**
3.2
**OS/device including version:**
Win 10 1903
**Issue description:**
Node selection outline not shown when node is in locked state.
**Steps to reproduce:**
Add 2 control nodes to blank scene
Lock one of the nodes
The node that is locked will no longer display it's selection outline
The other (not locked) still will
**Minimal reproduction project:**
Blank project with above steps

If the locked layer could be outlined in grey, that would be great. It is helpful in scenarios where you want to better understand the layout of a UI.

If someone would point me in the direction of the files which would deal with this issue, then I could potentially look at it. I tried having a look myself but am not sure where the interaction is between clicking the lock button and the editor display.

**Comment by KoBeWi** [comment]
> If someone would point me in the direction of the files which would deal with this issue, then I could potentially look at it.

Check this
https://github.com/godotengine/godot/blob/c9de04b628e225e74aededb3dc0fa7b772090660/editor/plugins/canvas_item_editor_plugin.cpp#L3196

**Comment by EricEzaM** [comment]
Thanks @KoBeWi, I managed to get it done! Hopefully my first contribution :) Should I submit it as a PR to master or 3.2?

Updated _draw_selection() to:

1.  Get locked items by changing `selection = _get_edited_canvas_items(false, false)` to `... (true, false)`
2. Added an if switch to change the outline color to grey if the item is locked.
3. The pivot point is not drawn if the item is locked

Preview:
![Fr14KxN9cc](https://user-images.githubusercontent.com/41730826/79332427-2a362180-7f60-11ea-907b-cfc178e212bd.gif)
